spec/request/gi/institutions_request_spec.rb

```diff
@@ -16,7 +17,7 @@
 
   it 'responds to GET #search with bad encoding' do
     VCR.use_cassette('gi_client/gets_search_results') do
-      get URI.encode("/v0/gi/institutions/search?name=\255illinois".force_encoding('ISO-8859-1'))
```

Comment:
Was this an autocrrect or was this manually tested?

Reply:
good call. I manually tested this. I tried to use the recommendations from the cop, but CGI.encode and alternatives don't work here. So instead, I took the result of that and copied/pasted. I think since it's a spec, it's pretty irrelevant one way or the other. The tests pass, thats all that matters, right? ;)

Reply:
I would be happy to add this to the Exclude list and leave it as it was. But I'm not entirely sure what we're trying to test here, parsing of URI?

Reply:
This was to test sentry errors that we were seeing with GIBCT where it would freak out about invalid character encodings in UTF-8 in the request params. If the test as modified still generates a bad request response then I guess it is still testing that case effectively.

Reply:
My guess is that the actual issue is caused by people copying/pasting institution names from weird word docs on old windows machines or something.

Reply:
Thanks @patrickvinograd. And yes, I basically took the result of what the url encoding returned in the console and checked to see that it matches when I copied and pasted. Should be the same test.